app/models/school.rb

```diff
@@ -1,5 +1,6 @@
 class School < ApplicationRecord
   SEARCH_RESULTS_LIMIT = 50
+  SEARCH_NOT_ENOUGH_CHARACTERS_ERROR = "search_term must have a minimum of 4 characters"
```

Comment:
👍 We should probably freeze this as well.

Reply:
Oops. Good spot. A thing I relied on Rubocop sorting out :).